syncp.go

```diff
@@ -215,8 +219,9 @@ func (r *conncount) ResetLen(n int) {
 }
 
 type connretry struct {
-	m map[conn]*retry
-	n int
+	m         map[conn]*retry
+	n         int
+	Redirects atomic.Uint64
```

Comment:
Remember to reset its value to 0 when we put the struct back into the sync.Pool.

```go
func (r *connretry) ResetLen(n int) {
	clear(r.m)
	r.Redirects.Store(0) // <- remember to do this
}
```

Reply:
I missed it. Thank you!
[f04b433](https://github.com/redis/rueidis/pull/643/commits/f04b433a99cf7b644720617ea5e4c5457ad9882b)